--- stratum/hal/bin/bmv2/BUILD  2022-08-02 09:39:11.678511803 +0000
+++ stratum/hal/bin/bmv2/BUILD.new      2022-08-02 09:40:20.572309119 +0000
@@ -90,9 +90,9 @@
     architecture = "amd64",
     data = ":stratum_bmv2_data",
     depends = [
-        "libboost-filesystem1.62.0",
-        "libboost-program-options1.62.0",
-        "libboost-thread1.62.0",
+        "libboost-filesystem1.67.0",
+        "libboost-program-options1.67.0",
+        "libboost-thread1.67.0",
         "libgmp10",
         "libjudydebian1",
         "libpcap-dev",